### PR TITLE
searchJob tests part 2 - users application status

### LINKED ISSUE #60: Test the user application function - searchJob feature

### PR DESCRIPTION
**Pull request by itaiv95**
testing the user application status functioning on multiple cases

Fixed: #60 

@benipeled @guy9050 @bardielle

### CODE REVIEW

File: searchJob/tests_2.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+
+def deleteAllobjects():
+    JobPost.objects.all().delete()
+    JobType.objects.all().delete()
+    JobCity.objects.all().delete()
+    Company.objects.all().delete()
+    User.objects.all().delete()
+    UserAccount.objects.all().delete()
+    UserApplication.objects.all().delete()
```
Comment (guy9050) [inline searchJob/tests_2.py]: if there is other data in DB wouldn't this delete everything? I think it's better to delete specifically the objects you create (wdyt? @bardielle @benipeled)

Comment (bardielle) [reply searchJob/tests_2.py]: Agree with you
Please open an issue or fix it as part of that PR

Comment (itaiv95) [reply searchJob/tests_2.py]: Hi, I tried it with manual data insertion Through the ui, and it didn't delete all the data. 
i will try again and try to fix that.
ty

Comment (bardielle) [reply searchJob/tests_2.py]: @itaiv95 any update?

Comment (itaiv95) [reply searchJob/tests_2.py]: Hi, yes I checked that and it doesn’t delete all the data- only the test data
@bardielle

Comment (guy9050) [reply searchJob/tests_2.py]: @itaiv95 how does it delete only the test data if you do "all().delete" without any filter?

Comment (bardielle) [reply searchJob/tests_2.py]: Of course it's not- when you run a test it creates a local env that simulator a DB and not running against on the real DB..
Still - you need to remove explicit rows...

Comment (guy9050) [reply searchJob/tests_2.py]: @itaiv95 ^

Comment (itaiv95) [reply searchJob/tests_2.py]: Ok, thank you i will fix and update it today
Thank you both!

Comment (itaiv95) [reply searchJob/tests_2.py]: done 
@guy9050 @bardielle

**Review by bardielle — APPROVED** [review]
/lgtm

**Pull request merged**